tests/test_client.py

```diff
@@ -1317,29 +1320,38 @@ def update_roles_v1(self, obj):
 
     def update_modifier_list(self, obj, version):
         obj_id = obj.get('id')
+        modifier_list_data = obj.get('modifier_list_data')
+        modifier_list_data['name'] = self.make_id('modifier_list')
         body = {'batches': [{'objects': [{'id': obj_id,
                                           'type': 'MODIFIER_LIST',
                                           'version': version,
-                                          'modifier_list_data': {'name': self.make_id('modifier_list')}}]}],
+                                          'modifier_list_data': modifier_list_data}]}],
                 'idempotency_key': str(uuid.uuid4())}
         return self.post_catalog(body)
 
-    def update_discounts(self, obj_id, version):
+    def update_discounts(self, obj, version):
+        obj_id = obj.get('id')
+        discount_data = obj.get('discount_data')
+        discount_data['name'] = self.make_id('discount')
+        discount_data['amount_money'] = {'amount': 34500,
+                                         'currency': 'USD'}
         body = {'batches': [{'objects': [{'id': obj_id,
                                           'type': 'DISCOUNT',
                                           'version': version,
-                                          'discount_data': {'name': self.make_id('discount'),
-                                                            'discount_type': 'FIXED_AMOUNT',
-                                                            'amount_money': {'amount': 34500,
-                                                                             'currency': 'USD'}}}]}],
+                                          'discount_data': discount_data,}]}],
                 'idempotency_key': str(uuid.uuid4())}
         return self.post_catalog(body)
 
-    def update_taxes(self, obj_id, version):
+    def update_taxes(self, obj, version):
+        obj_id = obj.get('id')
+        LOGGER.info('THIS IS WHAT THE TAX RECORD LOOKS LIKE %s', obj)
```

Comment:
Do we need to log this?

Reply:
Nope I can remove, was using for debugging at one point